templates/org/settings/options_dangerzone.tmpl

```diff
@@ -25,6 +35,48 @@
 	</div>
 </div>
 
+<div class="ui small modal" id="change-visibility-org-modal">
+	<div class="header">
+		{{ctx.Locale.Tr "org.settings.change_visibility"}}
+	</div>
+	<div class="content">
+		<ul class="ui warning message">
+			<li>{{ctx.Locale.Tr "org.settings.change_visibility_notices_1"}}</li>
+			<li>{{ctx.Locale.Tr "org.settings.change_visibility_notices_2"}}</li>
+		</ul>
+		<form class="ui form form-fetch-action" action="{{.Link}}/visibility" method="post">
+			{{.CsrfTokenHtml}}
+
+			<div class="field" id="visibility_box">
+				<label for="visibility">{{ctx.Locale.Tr "org.settings.visibility"}}</label>
```

Comment:
Why `for`

Reply:
475f40da1f62a0811548ed60d736ea849332921a